storage-account.tf

```diff
@@ -4,221 +4,207 @@ locals {
   mgmt_network_name    = "core-cftptl-intsvc-vnet"
   mgmt_network_rg_name = "aks-infra-cftptl-intsvc-rg"
 
-  sa_aat_subnets = [
-    data.azurerm_subnet.jenkins_subnet.id,
-    data.azurerm_subnet.aks-00-mgmt.id,
-    data.azurerm_subnet.aks-01-mgmt.id,
-    data.azurerm_subnet.aks-00-infra.id,
-    data.azurerm_subnet.aks-01-infra.id,
-    data.azurerm_subnet.aks-00-preview.id,
-  data.azurerm_subnet.aks-01-preview.id]
-
-  sa_other_subnets = [
+  preview_vnet_name           = "core-preview-vnet"
+  preview_vnet_resource_group = "aks-infra-preview-rg"
+  aks_env                     = var.env == "sandbox" ? "sbox" : var.env
+
+  aat_cft_vnet_name           = "cft-aat-vnet"
+  aat_cft_vnet_resource_group = "cft-aat-network-rg"
+
+  app_aks_network_name    = var.env == "sbox" || var.env == "perftest" || var.env == "aat" || var.env == "ithc" ? "cft-${local.aks_env}-vnet" : "core-${local.aks_env}-vnet"
+  app_aks_network_rg_name = var.env == "sbox" || var.env == "perftest" || var.env == "aat" || var.env == "ithc" ? "cft-${local.aks_env}-network-rg" : "aks-infra-${local.aks_env}-rg"
```

Comment:
var.env shouldn't be checking for "sandbox" instead of "sbox"? or aks_env variable can be used?